charts/iom/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.1.0
+version: 2.2.0
```

Comment:
That change is done automatically when the pull request is merged into main. Provided that the pr has label minor, major etc. If you changed it manually, it would fall because it would bump to 2.3.0 or the current version is not matching.

Reply:
Disabled bumping of version. It's more confusing than helpful in the current process and documentation process. Maybe I will change the documentation process later.